simapp/updated_config.toml

```diff
@@ -0,0 +1,10 @@
+[store]
```

Comment:
curious of why this toml is defined here?

Reply:
As a mistake, thanks for catching this.